api/v1/controllers/AuthController.js

```diff
@@ -70,8 +80,85 @@ function refreshToken (req, res, next) {
 		});
 }
 
+function requestPasswordReset (req, res, next) {
+	var userEmail = req.query.email;
+
+	UserService
+		.findUserByEmail(userEmail)
+		.then(function (user){
+			return AuthService.generateToken(user, scopes.AUTH);
+		})
+		.then(function (tokenVal){
+			//TODO: Determine exact url route for reset page
+			tokenURL = (config.isDevelopment ? "localhost:8080/" : "hackillinois.org/") + "v1/auth/resetpage?token=" + tokenVal;
```

Comment:
Can we make a top-level config variable that selects the right URL for us? Same with the reset page. It'd be great if I could just say `config.domain` to get `host:port` on development or `hackillinois.org` on production, just as it'd be great to say `config.uri.passwordReset` to get `v1/auth/resetpage?token=`